manala.npm/README.md

```diff
@@ -32,13 +32,19 @@ Using ansible galaxy requirements file:
 
 ## Role Variables
 
-| Name                  | Default | Type  | Description       |
-| --------------------- | ------- | ----  | ----------------- |
-| `manala_npm_packages` | [ ]     | Array | Npm packages list |
+| Name                  | Default | Type    | Description                                                         |
+| --------------------- | ------- | ------- | ------------------------------------------------------------------- |
+| `manala_npm_packages` | [ ]     | Array   | Npm packages list                                                   |
+| `manala_npm.update`   | false   | Boolean | Update to the latest version of npm                                 |
+| `manala_npm.state`    | -       | String  | The version of npm, could be `latest` to always have the latest one |
```

Comment:
Wait, where the hell this variable comes from ?

Reply:
My bad, thought you could overwrite it from there https://github.com/manala/ansible-roles/blob/master/manala.npm/tasks/packages.yml#L6

Reply:
I'll remove them then